### ISSUE TITLE
关于模型的输出，还有年龄和性别的损失函数

### ISSUE THREAD

**Issue by Cong-Wan** [Nebula4869/PyTorch_gender_age_estimation#1]
博主你好，我想问下关于这个模型只输出一个x，你是把这个输出当成年龄和性别合集了嘛，那为什么是性别的输出是
`logits[:, :2]`，
然后age的loss计算我没太搞懂为什么是这样算的， 还有总体的loss
` age_loss = torch.sum((logits[:, 2] - ages) ** 2 / weights[torch.clip(logits[:, 2].long() - 15, 0, 57)]) / len(logits)
  #age_loss = mse_loss(logits[:, 2], ages)

  loss = gender_loss + age_loss * 100`
如果你有空的话可以帮我解答一下嘛。谢谢！
我的邮箱是： Wilburwan@gmail.com，你也可以在这里回复我！
